x-pack/plugins/security_solution/server/endpoint/routes/actions/list_handler.test.ts

```diff
@@ -96,8 +89,10 @@ describe('Action List Handler', () => {
     });
 
     it('should return `badRequest` when sentinel_one feature flag is not enabled and agentType is `sentinel_one`', async () => {
-      // @ts-expect-error We're writing to a readonly property just for the purpose of the test
-      endpointAppContextService.experimentalFeatures.responseActionsSentinelOneV1Enabled = false;
+      apiTestSetup.endpointAppContextMock.experimentalFeatures = {
```

Comment:
The changes in this file were required, because previously the 
`endpointAppContextService.experimentalFeatures.responseActionsSentinelOneV1Enabled = false;` wasn't really overwriting the feature flag's value. It was using the real config value.